### PR TITLE
🤝🏻 Connection To Slack Via WebAPI

### LINKED ISSUE #1: Slack
connection to slack via webAPI.

### PR DESCRIPTION
**Pull request by AyalyBresler**
Create an automatic connection list to Slack via the WEB API.

closed #1

### CODE REVIEW

**Comment by gittylitman** [comment]
Where is the `requirements` file? where is the `.env.example` ? where is the tests?

**Comment by AyalyBresler** [comment]
> Where is the `requirements` file? where is the `.env.example` ? where is the tests?

The `requirements` file not exist? The `.env.example` not exist?

**Comment by AyalyBresler** [comment]
> > Where is the `requirements` file? where is the `.env.example` ? where is the tests?
> 
> The `requirements` file not exist? The `.env.example` not exist?

**Comment by gittylitman** [comment]
Add `.gitignore` file

File: Readme.md
Comment (gittylitman) [inline Readme.md:1]: The use of the application is not detailed enough.

File: src/slack_connection.py
Comment (gittylitman) [inline src/slack_connection.py:1]: What's the point of catching the error in order to throw it away?

Comment (gittylitman) [reply src/slack_connection.py:1]: You didn't answer, what's the point of throwing an error?

Comment (AyalyBresler) [reply src/slack_connection.py:1]: To manage them and send the user messages accordingly!

**Comment by AyalyBresler** [comment]
I waiting to review!
Are you waiting for me?

File: Readme.md
Comment (gittylitman) [inline Readme.md:1]: Provide more details on how to operate the system.

Comment (gittylitman) [inline Readme.md:1]: Give the commands to execute.

Comment (gittylitman) [reply Readme.md:1]: Give me the commands to execute in the docker.

**Review by gittylitman — APPROVED** [review]

**Comment by AyalyBresler** [comment]
Thank you so so much about all the review in the PR!!

________________________________
מאת: Gitty Litman ***@***.***>
‏‏נשלח: יום חמישי 12 דצמבר 2024 14:26
‏‏אל: AyalyBresler/slack ***@***.***>
עותק: a. Gershtenkoren ***@***.***>; State change ***@***.***>
‏‏נושא: Re: [AyalyBresler/slack] 🤝🏻 Connection To Slack Via WebAPI (PR #2)


Merged #2<https://github.com/AyalyBresler/slack/pull/2> into master.

**Pull request merged**